src/server/app.js

```diff
@@ -196,10 +197,19 @@ io.sockets.on('connection', function (socket) {
           socket.emit('update-feedback-interval', [resultNeg, resultPos])
         })
       })
-      // Get last 10 messages
-      messagesController.getLastTen(lecture).then(function (result) {
-        socket.emit('last-ten-messages', result.reverse())
+      // Get all messages
+
+      // If lecturer
+      // TODO
+      if (true) {
+        messagesController.getInPrioritizedOrder(lecture).then(function (result) {
+          socket.emit('all-messages-prioritized', result.reverse())
+        })
+      } else {
+        messagesController.getAllMessages(lecture).then(function (result) {
+        socket.emit('all-messages', result.reverse())
       })
```

Comment:
This is a mismatch on where sorting of messages should be performed. With the latest commits, sorting is now handled in front-end (for React efficiency), so this will be both redundant (but the function is not yet in use anyway).